src/wireviz/Harness.py

```diff
@@ -185,10 +202,22 @@ def create_graph(self) -> Graph:
             wirehtml.append('<table border="0" cellspacing="0" cellborder="0">')  # conductor table
             wirehtml.append('   <tr><td>&nbsp;</td></tr>')
 
-            for i, connection_color in enumerate(cable.colors, 1):
+            for i, (connection_color, wirelabel) in enumerate(zip_longest(cable.colors, cable.wirelabels), 1):
                 wirehtml.append('   <tr>')
                 wirehtml.append(f'    <td><!-- {i}_in --></td>')
-                wirehtml.append(f'    <td>{wv_colors.translate_color(connection_color, self.color_mode)}</td>')
+                wirehtml.append(f'    <td>')
+
+                wireinfo = []
+                if cable.show_wirenumbers:
+                    wireinfo.append(str(i))
+                colorstr = wv_colors.translate_color(connection_color, self.color_mode)
+                if colorstr:
+                    wireinfo.append(colorstr)
+                if cable.wirelabels:
+                    wireinfo.append(wirelabel if wirelabel else '')
```

Comment:
```suggestion
                    wireinfo.append(wirelabel if wirelabel is not None else '')
```
Otherwise, you might risk hiding labels that has non-string values that evaluates to False, e.g. `0`, `False`, `0.0`, `0j`, etc.